lib/local_inventory.rb

```diff
@@ -54,6 +53,7 @@ def save
       @klass.collection.insert(pending_inserts.map(&:as_document)) unless pending_inserts.empty?
       # Iterate over modified items and save
       (@updates - pending_inserts).each{|item|
+        Infrastructure.where(remote_id: item.remote_id).map {|i| i.update_attributes(record_status: 'updated')} if item.class == Infrastructure
```

Comment:
maybe it could be easier to replace "item.class == Infrastructure" with "item.is_a?(Infrastructure)", I think that this could be easier to read.